cmd/nexd/flags_unix.go

```diff
@@ -10,13 +10,16 @@ import (
 )
 
 var stateDirDefault = "/var/lib/nexd"
+var stateDirDefaultExpression = "/var/lib/nexd"
 
 func init() {
 
 	currentUser, err := user.Current()
 	if err == nil && currentUser.Uid != "0" && currentUser.HomeDir != "" {
 		stateDirDefault = filepath.Join(currentUser.HomeDir, ".nexodus")
+		stateDirDefaultExpression = "$HOME/.nexodus"
 		api.UnixSocketPath = filepath.Join(stateDirDefault, "nexd.sock")
+		api.UnixSocketPathExpression = "$HOME/.nexodus/nexd.sock"
```

Comment:
This looks good

Now I realize we're generating the docs for non-root mode, which probably isn't the default we want to include in our docs. I'll address that in a future patch though